tbx/project_styleguide/templates/patterns/pages/events/events_listing.yaml

```diff
@@ -3,6 +3,7 @@ context:
     strapline: Events
     title: Events
     get_verbose_name: events-listing
+    no_events_message: <p>We don't have any live events for you to register for at the moment but there will be more coming soon!</p>
```

Comment:
I don't think this has any immediate effect as we have some events defined in the yaml, but good to have it here for completeness - and I guess it allows you to comment out the events and test it.